helm/ambassador-agent/templates/clusterrole.yaml

```diff
@@ -62,7 +62,7 @@ metadata:
 rules:
 - apiGroups: [""]
   resources: [ "configmaps", "secrets" ]
-  verbs: [ "get", "list", "watch" ]
+  verbs: [ "get", "watch" ]
```

Comment:
I have the feeling that because of the api key, this should stil be list, but only in the installation namespace (so simple role, not cluster role).